src/patternfly/components/OptionsMenu/options-menu-menu-item.hbs

```diff
@@ -1,5 +1,5 @@
-<li>
-  <button class="{{pfv}}options-menu__menu-item{{#if options-menu-menu-item--modifier}} {{options-menu-menu-item--modifier}}{{/if}}"
+<li role="none">
+  <button role="option" class="{{pfv}}options-menu__menu-item{{#if options-menu-menu-item--modifier}} {{options-menu-menu-item--modifier}}{{/if}}"
```

Comment:
@thatblindgeye that would be fine with me. If the react markup is good, I think we should update the core markup since it's just documentation and less disruptive to users.

Reply:
@mcoker @thatblindgeye I do not think we should update the react implantation unless it is a critical bug reported by a consumer since the component is deprecated.